awx/main/utils/insights.py

```diff
@@ -2,42 +2,46 @@
 # All Rights Reserved.
 
 
+# Old Insights API -> New API
+#
+# last_check_in -> checked_on
+# reports[] -> active_reports[]
+# reports[].rule.{description,summary} -> active_reports[].rule.{description,summary}
+# reports[].rule.category -> active_reports[].rule.category.name
+# reports[].rule.severity (str) -> active_reports[].rule.total_risk (int)
+
+# reports[].rule.{ansible,ansible_fix} appears to be unused
+# reports[].maintenance_actions[] missing entirely, will be provided
```

Comment:
@jbradberry this is what we've been told will be provided within the next sprint?

Reply:
Yeah